src/styles/component/editnotes.css

```diff
@@ -0,0 +1,86 @@
+.edit-note-background{
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-color: rgba(0, 0, 0, 0.600);
+    place-items: center;
+   display: none;
+    z-index: 2; 
+}
+
+.edit-note-container{
+    border: 2px solid black;
+    width: 600px;
+    min-height: 5rem;
+    max-height: max-content;
+    gap: 1rem;
+    background-color: var(--white);
+    font-family: var(--montserrat-font);
+    box-shadow: var(--box-shadow-1);
+    border-radius: var(--radius-md);
+}
+
+.edit-note-container i{
+    cursor: pointer;
+    border-radius: 50%;
+    padding: 5px;
+    font-size: 2rem;
+}
+.edit-note-container i:hover{
+    color: var(--black);
+    background-color: var(--grey-lightest);
+}
+.top-section i{
+  margin-left: auto;
+}
+
+.edit-title{
+    font-size: 1.6rem;
+    padding: 0.2rem;
+}
+
+.edit-title:focus{
+    border: 2px solid var(--primary);
+}
+.edit-body{
+    font-size: 1.2rem;
+}
+
+
+.action-buttons{
+    margin-left: auto;
+    gap: 1rem;
+    cursor: pointer;
+}
+.show-edit-box{
+    display: grid;
+}
+.hide-edit-box{
+    display: none;
+}
+.error-text{
+    color: red
+}
+textarea, textarea:focus
```

Comment:
ideally do not directly mutatethe HTML tag, instead apply the styles by classname on textarea

Reply:
Done changes in new commit